core/tests/test_utils.py

```diff
@@ -143,16 +148,43 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Returns a string for unicode compatibility test.
+
+        Args:
+            suffix: string. Appended to unicode string.
```

Comment:
Types should be what you get when you run `type(suffix)` in Python. So, str or unicode (I'm not sure which). But not string.

Description could be: A suffix to append to the unicode string. (It's a bit nicer to say what the thing is, rather than just what's done with it.)

Reply:
done

Reply:
done